public/index.html

```diff
@@ -33,7 +33,7 @@
   <script src="//cdn.jsdelivr.net/react/15.4.2/react.min.js"></script>
   <script src="//cdn.jsdelivr.net/react/15.4.2/react-dom.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/graphiql@0.11.11/graphiql.min.js"></script>
-  <script src="schema.js"></script>
```

Comment:
@acao it will break local version or any non-official deployment.
The correct fix would be to configure netlify to redirect `https://graphql.org/swapi-graphql` => `https://graphql.org/swapi-graphql/`

Reply:
https://github.com/graphql/graphql.github.io/compare/master...acao:swapi-asset-redirect-fix?expand=1